src/gui/EffectView.cpp

```diff
@@ -257,7 +257,7 @@ void EffectView::paintEvent( QPaintEvent * )
 	QPainter p( this );
 	p.drawPixmap( 0, 0, m_bg );
 
-	QFont f = pointSize(font(), 7.5f);
+	QFont f = adjustedToPixelSize(font(), 10);
```

Comment:
Maybe this should be `p.font()` ? Depending on if it's needed like the others.

Reply:
The code already was like this before. I guess the idea is to fetch the unchanged widget font on only make some adjustments, e.g. to the font size.